daemon/cluster/executor/container/container.go

```diff
@@ -718,6 +718,36 @@ func (c *containerConfig) applyPrivileges(hc *enginecontainer.HostConfig) {
 			hc.SecurityOpt = append(hc.SecurityOpt, "label=type:"+selinux.Type)
 		}
 	}
+
+	// variable to make the lines shorter and easier to read
+	seccomp := privileges.Seccomp
+	if seccomp != nil {
+		switch seccomp.Mode {
+		case api.Privileges_SeccompOpts_DEFAULT:
+			// TODO(dperny): DO NOT MERGE WITHOUT THIS QUESTION ANSWERED!
```

Comment:
Should this PR be converted to draft until this question is answered?

Reply:
This review comment is why I put this code comment. Knew I would forget about it.